src/components/editor/State.js

```diff
@@ -2363,6 +2383,27 @@ class Observation extends Component<Props> {
     }
   }
 
+  renderCodeOrValueSetForValueCode() {
+    let state = ((this.props.state: any): ObservationState);
+     if (state.value_code.system) {
+       return (
+         <div className='section'>
+           Code <a className='editable-text' onClick={() => {this.props.onChange('value_code')({val: {id: {url: '', display: ''}}})}}>Add ValueSet</a>
+           <br />
+           <Code code={state.value_code} system={"SNOMED-CT"} onChange={this.props.onChange('value_code')} />
+         </div>
+       );
+     } else {
+       return (
+         <div className='section'>
+           <a className='editable-text' onClick={() => {this.props.onChange('value_code')({val: {id: null}}); this.props.onChange('value_code')({val: {id: getTemplate('Type.Code.Snomed')}}); }}>Add Code</a> ValueSet
+           <br />
+           <ValueSet valueSet={state.value_code} onChange={this.props.onChange('value_code')} />
+         </div>
+       );
+     }
+   }
+
   renderToggles(currentItem) {
     console.log(currentItem);
```

Comment:
It doesn't look like you actually added this??? Either way let's get this outta here ;)